src/Karma.sol

```diff
@@ -103,6 +146,25 @@ contract Karma is Initializable, ERC20Upgradeable, UUPSUpgradeable, AccessContro
         _removeRewardDistributor(distributor);
     }
 
+    /**
+     * @notice Sets the slash percentage for the contract.
+     * @dev Only the admin configure the slash percentage
+     * @param percentage The percentage to set (in basis points: 1% = 100, 10% = 1000, 100% = 10000)
```

Comment:
This is ok, but might be counter intuitive. Why not specifically accepting only 0-100? There would be a case where its needed something like 10.01% or 0.50%?

Reply:
Ah, that's simply for better precision and other protocols in Defi seem to go with that as well (Uniswap, Aave etc). I think it's also more future proof, as it's very easy to introduce higher precision percentages.